src/index.html

```diff
@@ -14,8 +14,55 @@
     >
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto&display=swap">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="mainCard" data-qa="card">
+      <img class="mainCard__photo"
+        src="images/imac.jpeg" alt="mac">
+
+      <div class="mainCard__description">
+        <div class="mainCard__model">
+          APPLE A1419 iMac 27" Retina 5k Monoblock (MNED2UA/A)
+        </div>
+        <div class="mainCard__details">
+          Product code: 195434
+        </div>
+      </div>
+
+      <div class="mainCard__review">
+        <div class="mainCard__stars">
```

Comment:
this `div` should have a modifier `stars--4` like you have in `stars` task

Reply:
but the issue of not passing the test is not due to this